pkg/controller/download_controller.go

```diff
@@ -148,7 +148,7 @@ func (c *downloadController) processDownload(key string) error {
 				cancel()
 			},
 			OnStoppedLeading: func() {
-				log.Debug("Processed Download.")
+				log.Info("Processed Download.")
```

Comment:
Which object did we process the download for?

Reply:
Same above.